modules/grafana-agent/telemetry-to-lgtm/README.md

```diff
@@ -0,0 +1,67 @@
+# Grafana Agent Telemetry to LGTM Stack Module
+
+Module to forward the Grafana Agent's own telemetry data to Loki, Mimir and Tempo stacks locally or in GrafanaCloud.
+
+## Agent Version
+
+`>= v0.33`
+
+## Module arguments
+
+The following arguments are supported when passing arguments to the module
+loader:
+
+| Name | Type | Description | Default | Required
+| ---- | ---- | ----------- | ------- | --------
+| `grafana_agent_port`    | `string`   | The port the Grafana Agent is running on. | `"12345"` | no
+| `grafana_agent_logpath` | `string`   | The filepath to Grafana Agent logs | | yes
+| `prometheus_endpoint`   | `receiver` | The Prometheus remote write endpoint. | | yes
+| `prometheus_user`       | `string`   | The Prometheus remote write basic auth username. | | yes
+| `prometheus_password`   | `secret`   | The Prometheus remote write basic auth password. | | yes
+| `loki_endpoint`         | `string`   | Loki endpoint | | yes
+| `loki_user`             | `string`   | Loki basic auth username. | | yes
+| `loki_password`         | `secret`   | Loki basic auth password. | | yes
+| `tempo_endpoint`        | `string`   | Tempo Endpoint | | yes
+| `tempo_user`            | `string`   | Tempo basic auth username. | | yes
+| `tempo_password`        | `secret`   | Tempo basic auth password. | | yes
+
+Grafana Agent logs must be forwarded to the file at `grafana_agent_logpath`.
```

Comment:
Thoughts on adding a quick snippet on how to do that?

Reply:
good call, will do